metadata.yaml

```diff
@@ -13,7 +13,7 @@ description: |
    This charm packages Apache Spark's History Server for Kubernetes environments."
 
 containers:
-  spark-history-server:
+  spark-history-server-k8s:
```

Comment:
Uhm, this is probably not needed. 

Try to stick to how other repos are dealing with this:

https://github.com/canonical/kafka-k8s-operator/blob/main/metadata.yaml#L20

Reply:
Actually here https://github.com/canonical/kafka-k8s-operator/blob/main/metadata.yaml#L13

Reply:
:+1: